tests/components/prometheus/test_init.py

```diff
@@ -64,55 +68,37 @@ def test_view(prometheus_client):  # pylint: disable=redefined-outer-name
     assert len(body) > 3
 
     assert "# HELP python_info Python platform information" in body
-    assert (
-        "# HELP python_gc_objects_collected_total "
-        "Objects collected during gc" in body
-    )
-
-    assert (
-        'temperature_c{domain="sensor",'
-        'entity="sensor.outside_temperature",'
-        'friendly_name="Outside Temperature"} 15.6' in body
-    )
-
-    assert (
-        'battery_level_percent{domain="sensor",'
-        'entity="sensor.outside_temperature",'
-        'friendly_name="Outside Temperature"} 12.0' in body
-    )
-
-    assert (
-        'current_temperature_c{domain="climate",'
-        'entity="climate.heatpump",'
-        'friendly_name="HeatPump"} 25.0' in body
-    )
-
-    assert (
-        'humidity_percent{domain="sensor",'
-        'entity="sensor.outside_humidity",'
-        'friendly_name="Outside Humidity"} 54.0' in body
-    )
-
-    assert (
-        'sensor_unit_kwh{domain="sensor",'
-        'entity="sensor.television_energy",'
-        'friendly_name="Television Energy"} 74.0' in body
-    )
-
-    assert (
-        'power_kwh{domain="sensor",'
-        'entity="sensor.radio_energy",'
-        'friendly_name="Radio Energy"} 14.0' in body
-    )
-
-    assert (
-        'sensor_unit_sek_per_kwh{domain="sensor",'
-        'entity="sensor.electricity_price",'
-        'friendly_name="Electricity price"} 0.123' in body
-    )
-
-    assert (
-        'sensor_unit_u0xb0{domain="sensor",'
-        'entity="sensor.wind_direction",'
-        'friendly_name="Wind Direction"} 25.0' in body
-    )
+    assert ("# HELP python_gc_objects_collected_total "
+            "Objects collected during gc" in body)
+
+    assert ('temperature_c{domain="sensor",'
+            'entity="sensor.outside_temperature",'
+            'friendly_name="Outside Temperature"} 15.6' in body)
+
+    assert ('battery_level_percent{domain="sensor",'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4493809)